modules/activitypub/main_test.go

```diff
@@ -8,6 +8,10 @@ import (
 	"testing"
 
 	"code.gitea.io/gitea/models/unittest"
+
+	_ "code.gitea.io/gitea/models" // https://discourse.gitea.io/t/testfixtures-could-not-clean-table-access-no-such-table-access/4137/4
```

Comment:
```suggestion
	_ "code.gitea.io/gitea/models"
```

The comment doesn't make sense now, because there are a lot of such code